internal/authz/allow/allow.go

```diff
@@ -31,3 +31,7 @@ func (a *AllowAllAuthz) CreateTuples(ctx context.Context, r *kessel.CreateTuples
 func (a *AllowAllAuthz) DeleteTuples(ctx context.Context, r *kessel.DeleteTuplesRequest) (*kessel.DeleteTuplesResponse, error) {
 	return &kessel.DeleteTuplesResponse{}, nil
 }
+
+func (a *AllowAllAuthz) SetWorkspace(ctx context.Context, id int64, workspace, name, namespace string) (*kessel.CreateTuplesResponse, error) {
```

Comment:
I don't think we want to require the ID of the resource to be `int64` right?

We need the reporter's ID to be stored in relations graph, in addition to the inventory ID. As the inventory ID has no persistence for it today otherwise, it's not needed right now.

So in order to support the reporter's ID, which I don't believe we need to (or should) constrain, I think we want this to be an arbitrary string (essentially only limited by the resource ID constraints of the relations API which is very minimal, essentially no whitespace and a few other chars IIRC).